rtc-video-quality/encoder_commands.py

```diff
@@ -36,7 +36,7 @@ def aom_command(job, temp_dir):
     clip = job['clip']
     fps = int(clip['fps'] + 0.5)
     command = [
-      AOM_ENC_BIN,
+      binary_vars.binary_vars.AOM_ENC_BIN,
```

Comment:
You shouldn't need "binary_vars.binary_vars." here. Just one should suffice.

Reply:
Oh sorry about this mistake. I'll update it